books/free-programming-books-langs.md

```diff
@@ -1890,6 +1890,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [From Python to NumPy](https://www.labri.fr/perso/nrougier/from-python-to-numpy/) - Nicolas P. Rougier (3.6)
 * [Full Stack Python](https://www.fullstackpython.com) - Matt Makai
 * [Functional Programming in Python](https://www.oreilly.com/ideas/functional-programming-in-python) - David Mertz
+* [Fundamentals of Data Visualization](https://clauswilke.com/dataviz/) - Claus O. Wilke
 * [Fundamentals of  Python Programming](https://web.archive.org/web/20191005170430/http://python.cs.southern.edu/pythonbook/pythonbook.pdf) - Richard L. Halterman (PDF) (:construction: *in process*)
```

Comment:
Kindly Remove one space between 'of' and 'python' at line 1894, that is causing the alphabetical swap error, else your checks shall pass.

Reply:
Done